packages/obatcher/obatcher.1.0/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "A Framework for building Batched Concurrent Data Structures"
+maintainer: ["Lee Koon Wen"]
+authors: ["Lee Koon Wen" "Kiran Gopinathan" "Arthur Wendling"]
+license: "ISC"
+homepage: "https://github.com/koonwen/obatcher"
+doc: "https://koonwen.github.io/obatcher"
+bug-reports: "https://github.com/koonwen/obatcher/issues"
+depends: [
+  "dune" {>= "3.9"}
+  "ocaml" {>= "5.0.0"}
+  "picos" {= "0.5.0"}
```

Comment:
```suggestion
  "picos" {>= "0.5.0"}
```

We try not to upper bound packages unless necessary

Reply:
Thanks for the review! I had this constraint because picos is in early stages with breaking changes between versions. That said, I can probably monitor this package closely for the time being.